src/renderers/webvr/WebXRManager.js

```diff
@@ -215,18 +241,18 @@ function WebXRManager( renderer ) {
 
 	function onAnimationFrame( time, frame ) {
 
-		pose = frame.getDevicePose( frameOfReference );
+		pose = frame.getDevicePose !== undefined ? frame.getDevicePose( frameOfReference ) : frame.getViewerPose( frameOfReference );
 
 		if ( pose !== null ) {
 
 			var layer = session.baseLayer;
```

Comment:
Same here with
`var layer =  'renderState' in session ? session.renderState.baseLayer : session.baseLayer;`